src/Lib/Mail/Mail.php

```diff
@@ -0,0 +1,105 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Lib\Mail;
+
+class Mail
+{
+
+    /** @var string */
+    private $headders;
+
+
+    /**
+     *
+     * @param string $to
+     * @param string $from
+     * @param string $fromName
+     * @param string $content
+     */
+    public function __construct(string $to, string $from, string $fromName, string $subject, string $content)
+    {
+        $this->to = $to?$to:false;
```

Comment:
These variables should be private and there is missing check for valid email. Same lines 24, 25 and 26 should be private.

Reply:
you mean `private string $to`?

Reply:
Yes, define them before constructor.